guides/compensation/expenses/travel.md

```diff
@@ -2,15 +2,29 @@
 
 If you incur travel expenses as part of your work at Made Tech, the business will reimburse your standard class travel costs.
 
-Some examples of expensable work travel:
+Some examples of work travel you can expense:
 
 - Public transport / taxi to a client site for a meeting
 - Taxi home after working late
 - Travelling to work at a client site outside of the region of your Made Tech office
 
 
+## Using your own car
+You can use your private car where you cannot or it's not practical to use public transportation to travel for business. The current reimbursement rate as prescribed by HMRC is 45p per mile (up to 10k miles).
```

Comment:
I think a link to the HMRC web page would be good here. I assume it's the following link - https://www.gov.uk/expenses-and-benefits-business-travel-mileage/rules-for-tax

If we do add the link, maybe the heading for this section can be updated to say "Using your own vehicle" instead(?).

Reply:
Great idea. Updated.